campaigns.json

```diff
@@ -2759,16 +2786,25 @@
     "go": false,
     "lumisize": -1,
     "maxcopies": 1,
-    "SecondaryLocation": [
-       "T1_DE_KIT_Disk",
-       "T1_US_FNAL_Disk"
-    ],
     "SiteBlacklist": [
        "T2_CH_CERN",
        "T2_CH_CERN_HLT"
      ],
     "secondaries": {
-       "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL17_106X_mc2017_realistic_v9-v1/PREMIX" : {}
+          "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL17_106X_mc2017_realistic_v9-v1/PREMIX" :
+          {
+                  "SecondaryLocation": [
+                     "T1_DE_KIT_Disk",
+                     "T1_US_FNAL_Disk"
+                      ]
+          },
+          "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL17CP5_106X_mc2017_realistic_v9-v2/PREMIX" :
+          
```

Comment:
@scarletnorberg  you're missing a curly bracket here. Once again, I'd suggest using a JSON format checker before submitting PRs ;)